byoeb-v1/byoeb/byoeb/services/chat/message_handlers/user_flow_handlers/process.py

```
@@ -205,11 +206,20 @@ async def handle(
-            })
-            span.update_trace(user_id=message.user.user_id)
+            assert message.message_context is not None
+            assert message.user is not None
+            span.update(input=message.message_context.message_source_text, metadata={"message_id": message.message_context.message_id})
+
+            tags = []
+            try:
+                tags.append(LanguageCode(message.user.user_language).name.lower())
+            except ValueError:
+                pass
+            if message.user.user_type: tags.append(message.user.user_type)
```

Ruff (0.15.10)

[error] Multiple statements on one line (colon)

(E701)